src/conf/local-file.yaml

```diff
@@ -32,6 +32,7 @@ database:
   user: root
   password: password
   dbname: accuknox
+  sqlite-db-path: /home/vagrant/accuknox.db
```

Comment:
This cannot be used as the path since the /home/vagrant path might not be available on the target system. Lets use current directory for keeping accuknox.db